bin/compile

```diff
@@ -67,8 +67,11 @@ case "$stack" in
   "heroku-16")
     PACKAGES="libxss1 libx11-xcb1 libxcomposite1 libxcursor1 libxdamage1 libxfixes3 libxi6 libxtst6 libnss3 libgconf-2-4 libxrandr2 libasound2 libatk1.0-0 libgtk2.0-0 libgtk-3-0 libxinerama1 libcairo-gobject2"
     ;;
+  "heroku-18")
```

Comment:
Is it possible in bash to do something like `"heroku-16", "heroku-18")` so we don't have the duplication?